MANIFEST.in

```diff
@@ -4,3 +4,4 @@ recursive-include docs *.html *.css *.png
 recursive-include examples *.py *.csl *.bib
 recursive-include tests *.py *.bib
 prune tests/citeproc-test
+prune tests/local
```

Comment:
it should be included, not pruned per se. `tests/citeproc-test` seems to be some clone which tests do "behind the curtain".   I think you should instead add `*.txt` into line above  

```
recursive-include tests *.py *.bib *.txt
```

and please check if it does include those files in source distribution

Reply:
> it should be included, not pruned per se. `tests/citeproc-test` seems to be some clone which tests do "behind the curtain". I think you should instead add `*.txt` into line above

I've modified in dbbb5de.

> and please check if it does include those files in source distribution

Yes, the `tests/local/*.txt` files are included.